docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description:** The maximum number of threads that can be run by the MySQL server in the FE node to process I/O events.
+
+##### mysql_nio_backlog_num
+
+- **Default:** 1024
+- **Description:** The length of the backlog queue held by the MySQL server in the FE node.
+
+##### max_mysql_service_task_threads_num
+
+- **Default:** 4096
+- **Description:** The maximum number of threads that can be run by the MySQL server in the FE node to process tasks.
+
+##### mysql_server_version
+
+- **Default:** 5.1.0
+- **Description:** The MySQL server version returned to the client. Modifying this parameter will affect the version information in the following situations:
+  1. `select version();`
+  2. Handshake packet version
+  3. Value of the global variable `version` (`show variables like 'version';`)
+
+##### max_connection_scheduler_threads_num
+
+- **Default:** 4096
+- **Description:** The maximum number of threads that are supported by the connection scheduler.
+
+##### qe_max_connection
+
+- **Default:** 1024
+- **Description:** The maximum number of connections that can be established by all users to the FE node.
+
+##### check_java_version
+
+- **Default:** TRUE
+- **Description:** Specifies whether to check version compatibility between the executed and compiled Java programs. If the versions are incompatible, StarRocks reports errors and aborts the startup of Java programs.
+
+#### Metadata and cluster management
+
+##### meta_dir
+
+- **Default:** StarRocksFE.STARROCKS_HOME_DIR + "/meta"
+- **Description:** The directory that stores metadata.
+
+##### heartbeat_mgr_threads_num
+
+- **Default:** 8
+- **Description:** The number of threads that can be run by the Heartbeat Manager to run heartbeat tasks.
+
+##### heartbeat_mgr_blocking_queue_size
+
+- **Default:** 1024
+- **Description:** The size of the blocking queue that stores heartbeat tasks run by the Heartbeat Manager.
+
+##### metadata_failure_recovery
+
+- **Default:** FALSE
+- **Description:** Specifies whether to forcibly reset the metadata of the FE. Exercise caution when you set this parameter.
+
+##### edit_log_port
+
+- **Default:** 9010
+- **Description:** The port that is used for communication among the leader, follower, and observer FEs in the StarRocks cluster.
+
+##### edit_log_type
+
+- **Default:** BDB
+- **Description:** The type of edit log that can be generated. Set the value to `BDB`.
+
+##### bdbje_heartbeat_timeout_second
+
+- **Default:** 30
+- **Description:** The amount of time after which the heartbeats among the leader, follower, and observer FEs in the StarRocks cluster time out. Unit: second.
+
+##### bdbje_lock_timeout_second
+
+- **Default:** 1
+- **Description:** The amount of time after which a lock in the BDB JE-based FE times out. Unit: second.
+
+##### max_bdbje_clock_delta_ms
+
+- **Default:** 5000
+- **Description:** The maximum clock offset that is allowed between the leader FE and the follower or observer FEs in the StarRocks cluster. Unit: ms.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/877f1820-ef7b-4acd-874f-d8a611830684)
See the comment at Line 969